src/density-qvm.lisp

```diff
@@ -286,6 +286,17 @@ VEC-DENSITY and (perhaps freshly allocated) TEMPORARY-STORAGE."
          (aref vec-density (+ (* i dim) i)) ; this is rho[i,i]
          )))))
 
+(defun density-qvm-basis-probabilities (qvm)
+  "Computes a vector of probabilities for each element in the computational basis."
```

Comment:
Be more specific in this docstring. Also do a `check-type` that it's the right qvm.